portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,6 +53,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Gson gson = new Gson();
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(comments));
+
+    response.sendRedirect("/index.html");
```

Comment:
Is this needed?

Reply:
in my index.html I call the doGet function on the from with the dropdown, so that is why i included the redirect.

Reply:
But also I just realized it doesn't allow the comments to print on the page since I am redirecting twice, I will fix that